packages/form-js-editor/test/spec/properties-panel/PropertiesPanel.spec.js

```diff
@@ -74,6 +66,39 @@ describe('properties panel', function() {
     expect(result.container.querySelectorAll('.fjs-properties-panel-group')).to.have.length(3);
   });
 
+  it('should render (readonly)', async function() {

```

Comment:
Test cover on the viewer: 

* Import behavior (blow up on unsupported schema with a reasonable error message)